website/integrations/services/slack/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Slack
+---
+
+<span class="badge badge--secondary">Support level: authentik</span>
+
+## What is Slack
+
+> Slack is a platform for collaboration, with chat and real-time video capabilities.
+>
+> -- https://slack.com
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `slack.company` is the FQDN of the Slack install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik configuration
+
+### Step 1. Create custom property mappings
+
+Your Slack integration requires property mappings for `User.Email` and `User.Username` so that authetnik can retrieve and map these values from Slack.
+
+1. Log in as admin to your authentik instance and then click **Admin interface**.
+2. Navigate to **Customization -> Property Mappings** and then click **Create**.
+3. On the **New property mapping** modal, select **SAML Property Mapping** and then click **Next**.
+4. Define the required values. For **Expression** define `User.Email`, with `return request.user.email` and for `User.Username` use `return request.user.username`.
```

Comment:
It's not very clear that you need to create two of them.

Reply:
Yeah, fair point... I re-formatted that section to callout the seperate steps for each of the two property mappings. Let me know if it makes it more obvious now.